keymaps/atom-phpunit.json

```diff
@@ -3,7 +3,9 @@
     "alt-shift-t": "atom-phpunit:run-test",
     "alt-shift-c": "atom-phpunit:run-class",
     "alt-shift-s": "atom-phpunit:run-suite",
-    "alt-shift-r": "atom-phpunit:run-last-test",
-    "alt-shift-x": "atom-phpunit:toggle-output"
+    "alt-shift-r": "atom-phpunit:run-last-test"    
+  },
+  "atom-workspace atom-text-editor:not([mini])": {
+    "escape': 'atom-phpunit:toggle-output"
```

Comment:
The quotes are wrong here: all of the quotes should be double quotes ("), with no single quotes (')